nucliadb_protos/nodereader.proto

```diff
@@ -108,6 +108,7 @@ message ParagraphSearchRequest {
     bool with_duplicates = 14;
     bool only_faceted = 15;
     optional string advanced_query = 16;
+    repeated string keys = 17;
```

Comment:
What are this `keys`? Could we have a more descriptive name?

Reply:
AFAIK "keys is a core concept in the node" which is essentially a paragraph id.
I added the `_filter` suffix, so that at least it is clear that we use this attribute for filtering.

Reply:
I didn't know. If it is, that's okay for me